docs/DeveloperGuide.md

```diff
@@ -133,6 +158,51 @@ Guarantees: Editing of skill or achievement is successful
   1a1. InternHunter displays an error message and informs the user of the valid input format. </br>
   Use case resumes from step 1.
 
+
+### Use case: UC14 - Get help
+
+#### MSS
+
+Precondition: User needs help with using the system.
```

Comment:
shouldn't precondition be the state of the system and not the user?

Reply:
True, I should remove this, I think the only precondition then is that the system is running which is not very meaningful.